aria-practices.html

```diff
@@ -6583,12 +6593,12 @@ <h2>Structural Roles</h2>
       These roles express the meaning that is conveyed by the layout and appearance of elements that organize and structure content, such as headings, lists, and tables.
     </p>
     <p>
-      some host languages, such as HTML, include elements that express the same  semantics as an ARIA role. 
+      some host languages, such as HTML, include elements that express the same  semantics as an ARIA role.
```

Comment:
```suggestion
      some host languages, such as HTML, include elements that express the same semantics as an ARIA role.
```
Remove space after "same."